docs/Configuration/fleet-server-configuration.md

```diff
@@ -2896,6 +2896,40 @@ The duration between DEP device syncing (fetching and setting of DEP profiles).
   mdm:
     apple_dep_sync_periodicity: 10m
   ```
+##### fleet_mdm_windows_wstep_identity_cert_bytes
+The content of the Windows WSTEP identity certificate. An X.509 certificate, PEM-encoded.
+- Default value: ""
+- Environment variable: fleet_mdm_windows_wstep_identity_cert_bytes  
+- Config file format: 
+  ```
+  mdm:
+   fleet_mdm_windows_wstep_identity_cert_bytes: |
```

Comment:
```suggestion
   windows_wstep_identity_cert_bytes: |
```

For the config file format, we want to remove the `fleet_mdm` part because the option is just `windows_wstep_identity_cert_bytes`. You can see that it lives under `mdm`